gigantum/gigantum-client
# 1588 import modal

## Pull Request for Issue #1588 

**Describe changes introduced by this PR (Bugs fixed, features added, docs updated, etc.)**:
Adds blocking when server is in backup mode on the remote import modal. Handles failure and checks for backup mode.
Adds a modal that pops when backup mode has been detected. Modal can be dismissed for the session if the users chooses to dismiss it.


**Please check if the PR fulfills these requirements**

  - PR is set to merge to integration
  - PR title is descriptive, as this becomes the commit message on squash and merge.
  - Any related issues are added to the extended commit message during squash and merge.
  - Tests have been added (for bug fixes/features)
  - Docs have been added/updated (for bug fixes/features)

**QA Test Steps and Exit Criteria**:
1. load ui
2. change server to backup mode
3. try to import a project
4. import should fail and backup mode should lock down remote operations.
5. modal should pop saying 'back in progress'
6. reload modal should pop again
7. check the checkbox 'don't show me this again' and hit the x in the top right.
8. reload and the modal should popup, check the checkbox 'don't show me this again'  and hit dismiss
9. reload, modal should not popup this time
